modules/462-loki/templates/statefulset.yaml

```diff
@@ -49,7 +90,7 @@ spec:
             mountPath: /loki
         ports:
           - name: http-metrics
-            containerPort: 3100
```

Comment:
This port should not be accessible because the listen address should be 127.0.0.1.
 All communications with the loki should be provided only through kube-rbac-proxy.